data/OR/incentives.json

```diff
@@ -0,0 +1,962 @@
+[
+  {
+    "id": "OR-1",
+    "authority_type": "state",
+    "authority": "or-energy-trust-of-oregon",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "or_energyTrustOfOregon_energyTrustOfOregon",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 500
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$500 rebate for a ductless heat pump replacing zonal electric resistance as the primary heat source. Primary unit must be in the main living space."
+    },
+    "start_date": "2023"
+  },
+  {
+    "id": "OR-2",
+    "authority_type": "state",
+    "authority": "or-energy-trust-of-oregon",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "or_energyTrustOfOregon_energyTrustOfOregon",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 2000
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
```

Comment:
I think the `owner_status` for OR is frequently wonky. A portable heat pump maybe, but ductless *replacing electric resistance* is not something a renter can do.

It's a bit of a pain, but I think you can quickly redo this column in the spreadsheet by looking at the list in https://github.com/rewiringamerica/incentive-data-llm/blob/d56ac40232df4fb5f6f529584f82988e07383fef/src/constants.ts#L16. That has a fairly reliable mapping between the Technology type and owner_status (though it's a bit old so some Technologies may be missing).

I think it'll take ~15 minutes to do by hand, sorry.

Reply:
Ah okay, will do! I think they were coded this way because EnergyTrust specifies the eligible residence types (Owner Occupied vs. Rental Property). But even if a rental property is eligible, I agree that the renter won't be the decision maker.

If the technologies map to the owner status, could we just remove the owner status from the spreadsheets? (Out of scope for this PR of course, just wondering.)

Reply:
It's a good question. I feel like this came up once before but I can't remember when/in what forum.

I think it's pretty reasonable to make these properties of the underlying Item/Technology and take them off the individual incentives, but I'll defer to @oyamauchi and @RandomEtc.